v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
For the bare minimum of test coverage of the spec it would be good to have each part of ResourceRequirement used in a conformance test, even if we don't verify it.

:+1: for expected failing tests (and a third category of optional tests (for optional features) can be added at the same)